### PR TITLE
 Adding the User param to the site config so that it can be supported by Azure as an extra param

### PR DESCRIPTION
**Pull request by arafatkatze**
[See what the field Does
](https://learn.microsoft.com/en-us/answers/questions/1607830/how-does-the-user-parameter-work-in-azure-openai-s)

## Test plan
Tested locally by setting up extra variable in siteadmin with azure config and it worked perfectly.

### CODE REVIEW

File: internal/completions/client/azureopenai/openai.go [hunk 1]
```diff
@@ -288,6 +291,11 @@ func streamChat(
 			}
 			return nil
 		}
+		// These are only included in the last message, so we're not worried about overwriting
+		if entry.Usage != nil {
+			promptTokens = *entry.Usage.PromptTokens
+			fmt.Println("Prompt Tokens: ", promptTokens)
```
Comment (jamesmcnamara) [inline internal/completions/client/azureopenai/openai.go:297]: Is this meant to be left in?

Comment (arafatkatze) [reply internal/completions/client/azureopenai/openai.go]: cleaned up my testing mistakes

File: schema/site.schema.json [hunk 2]
```diff
@@ -2838,6 +2838,10 @@
           "description": "DEPRECATED. Use chatModel instead.",
           "type": "string"
         },
+        "user": {
+          "description": "Its the user field for OpenAI config for both AzureOpenAI and OpenAI",
```
Comment (dominiccooney) [inline schema/site.schema.json:2842]: Just "The ...", we don't need "Its ..."

(Its is possessive anyway and definitely wrong.)

**Review by dominiccooney — APPROVED** [review]
LGTM, I assume the ,omitempty is all we need to make this a no-op for models who do not use it and sites which do not specify it.

File: schema/site.schema.json (hunk 2 above)
Comment (arafatkatze) [reply schema/site.schema.json]: Fixed

**Pull request merged**